src/components/Disputes/PrecedenceCampaign/PrecedenceCampaignBanner.js

```diff
@@ -1,26 +1,40 @@
-import React, { useCallback, useState } from 'react'
+import React, { useCallback, useEffect, useRef, useState } from 'react'
 import { Button, GU, useTheme } from '@aragon/ui'
+
 import Modal from './PrecedenceCampaignModal'
+import StoredList from '../../../StoredList'
+import { getNetworkType } from '../../../lib/web3-utils'
+
+const KEY_PREFIX = 'PCDisputeBannerRead'
 
-const KEY_PREFIX = 'PC_DISPUTE_BANNER_READ'
+function getStoredList() {
```

Comment:
This is not a big issue, but I was wondering if we could just instantiate this once.

We currently use this same pattern of creating a new one on every call in the client, but I'm not sure if there was a reason we needed to do so @bpierre?

Reply:
No I don’t think it is needed, we could probably do it once :+1: 

```jsx
const storedList = new StoredList(`${KEY_PREFIX}:${getNetworkType()}`)
```

Reply:
Ah yeah true, since this is account agnostic we can use the same one 👍